website/views/shared/profiles/achievements.jade

```diff
@@ -146,8 +146,8 @@ div(ng-if=':: profile.achievements.originalUser')
   
 div(ng-if='profile.achievements.challenges || user._id == profile._id')
   // This is a very strange icon to use. revisit
-  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges')
-  div(ng-class='{muted: !profile.achievements.challenges}')
+  .achievement.achievement-karaoke(ng-show='profile.achievements.challenges.length')
```

Comment:
Thanks for the info! Would you like me to make those changes (`ng-show` -> `ng-if` and one time bindings) on all of the relevant achievements?